DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj

```diff
@@ -55,6 +46,7 @@
     <DefineConstants>DEBUG</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <MtouchLink>SdkOnly</MtouchLink>
```

Comment:
We may want link all here to test out Our linked on tests

Reply:
I see the device tests builds iOS as release, but Android and UWP as debug...

Reply:
Looks like this was the existing implicit default, so I guess it's ok!